one-to-one-scheduler/one-to-one-scheduler.gs

```diff
@@ -54,23 +61,26 @@ Version: ${LAST_PUBLISHED_VERSION + 1}
 
 - Your sheet must have the following columns in the given order
 
-1. Module Name - Name of the Module. To be filled by the team.
-2. Student Name - Name of the student. To be filled by the team.
-3. Student Email - Email id of the student. To be filled by the team.
-4. Date - Date of 1:1 in yyyy-mm-dd format. To be filled by the team.
-5. Start time - Start time of 1:1 in hh:mm format (24 hours). To be filled by the team.
-6. End time - End time of 1:1 in hh:mm format (24 hours). To be filled by the team.
-7. Meeting link - Auto populated by the script.
-8. Student response - Auto populated by the script.
-9. Ops Request - (0 - ignore the row),(1 - for meet invites),(2 - for Cancel Meeting) To be filled by the team.
-10. SSM emails - Additional emails you want to include in the meeting - Comma separated values.
-11. Mentor email - Email id of the mentor. To be filled by the team.
-12. Mentor response - Auto populated by the script.
+1. Session Type - (1:1 or Mock Interview). To be filled by the team.
+2. Module Name - Name of the Module/ Name of the Mock. To be filled by the team.
+3. Student Name - Name of the student. To be filled by the team.
+4. Student Email - Email id of the student. To be filled by the team.
+5. Date - Date of 1:1 in yyyy-mm-dd format. To be filled by the team.
+6. Start time - Start time of 1:1 in hh:mm format (24 hours). To be filled by the team.
+7. End time - End time of 1:1 in hh:mm format (24 hours). To be filled by the team.
+8. Meeting link - Auto populated by the script.
+9. Student response - Auto populated by the script.
+10. Ops Request - (0 - ignore the row),(1 - for meet invites),(2 - for Cancel Meeting) To be filled by the team.
+11. SSM emails - Additional emails you want to include in the meeting - Comma separated values.
+12. Mentor email - Email id of the mentor. To be filled by the team.
+13. Mentor response - Auto populated by the script.
+14. Recording Link - Auto populated by the script.
+15. Chat Link - Auto populated by the script.
 `;
 
 const DATE_DELIMITER = "-";
 
-const MEETING_TITLE_PREFIX = " - 1:1 (";
+const DEFAULT_SESSION_TYPE = "1:1";
```

Comment:
There is a naming convention here. The prefix must be `MEETING_`. You can name it as `MEETING_DEFAULT_TYPE`.

Reply:
Changed `DEFAULT_SESSION_TYPE` to `MEETING_DEFAULT_TYPE`